app/Steps.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App;
+
+use ReflectionClass;
+
+class Steps
+{
```

Comment:
When i have more brain power I'm going to take a look over this class. Feels like it can be simplified a little but I can't dig into it with a cursory review